### PR TITLE
Using a libjemalloc Allocator for Better Memory Management

### PR DESCRIPTION
**Pull request by goetzrrGit**
* **Tickets addressed:** AERIE-000
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description
We are still noticing a slow climb in the RSS for the sequencing server. @dandelany found this thread on Node about using a different allocator. https://github.com/nodejs/help/issues/1518#issuecomment-991798619

I have implemented the suggested fix and noticed during my 13 expansion runs with a Clippers plan, expansion logic, and dictionaries that the RSS doesn't climb past 3.6GB where as before I was getting past 4GB

### CODE REVIEW

**Comment by dandelany** [comment]
Related issue: #1476

**Review by skovati — COMMENTED** [review]
FYI, alpine does support jemalloc as well:
https://pkgs.alpinelinux.org/packages?name=jemalloc&branch=v3.17&repo=main&arch=&maintainer=

**Comment by Mythicaeda** [comment]
> FYI, alpine does support jemalloc as well: https://pkgs.alpinelinux.org/packages?name=jemalloc&branch=v3.17&repo=main&arch=&maintainer=

Documenting this here as well: I tried using the alpine version and didn't see it loaded when checking the node processes.

**Comment by goetzrrGit** [comment]
@Mythicaeda I tested this locally and verified that jemalloc is being used. Thank you for figuring it out. I can't approve this PR as it is my own but if you don't have any more updates feel free to approve it.

**Review by Mythicaeda — APPROVED** [review]

**Pull request merged**
